ext/pycall/pycall.c

```diff
@@ -1897,8 +1910,8 @@ pycall_pydict_from_ruby_iter(VALUE key, VALUE value, VALUE arg)
   if (res == -1) {
     return ST_STOP;
   }
-  Py_API(Py_DecRef)(pyobj_key);
-  Py_API(Py_DecRef)(pyobj_value);
+  pycall_Py_DecRef(pyobj_key);
+  pycall_Py_DecRef(pyobj_value);
   return ST_CONTINUE;
```

Comment:
Could you please rewrite by this?

```suggestion
  pycall_Py_DecRef(pyobj_key);
  pycall_Py_DecRef(pyobj_value);
  return (res == -1) ? ST_STOP : ST_CONTINUE;
```

Reply:
OK